channels/candidate-4.4.yaml

```diff
@@ -1,4 +1,7 @@
 name: candidate-4.4
 versions:
 - 4.3.5
+# No 4.3.6 because of https://bugzilla.redhat.com/show_bug.cgi?id=1811886
+# No 4.3.7 because of the service CA rotation issue: https://bugzilla.redhat.com/show_bug.cgi?id=1810036 https://bugzilla.redhat.com/show_bug.cgi?id=1801573
 - 4.4.0-rc.0
+# No 4.4.0-rc.1 because of the service CA rotation issue: https://bugzilla.redhat.com/show_bug.cgi?id=1810036
```

Comment:
This is a mistake, The 1810036 causes the issue for upgrades but for fresh installs there is no issue for 13months. So it should be in candidate but the edges should be blocked.

Reply:
> This is a mistake, The 1810036 causes the issue for upgrades but for fresh installs there is no issue for 13months.

I wasn't aware that RCs were hard to cut, but turns out they're ~weekly.  Updated to add 4.4.0-rc.1 with no incoming edges with db2267a53846.